data/data/libvirt/bootstrap/main.tf

```diff
@@ -13,7 +13,7 @@ resource "libvirt_ignition" "bootstrap" {
 resource "libvirt_domain" "bootstrap" {
   name = "${var.cluster_id}-bootstrap"
 
-  memory = "2048"
+  memory = "${var.bootstrap_memory}"
```

Comment:
this is invalid for the version of terraform used by the installer...
https://www.terraform.io/upgrade-guides/0-12.html#first-class-expressions

Reply:
ah..thanks for pointing this out. Just noticed that it succeeded even with the error before so i missed it.